doc/advanced_configuration.rst

```diff
@@ -554,4 +555,18 @@ in your own site's CSS file:
     }
 
 
+.. _min_reported_time:
```

Comment:
mind if I clean up this wording a small amount before merging? happy to do it myself as a quick commit-then-merge

Reply:
Feel free to force push to my branch.